packages/devtools_app/lib/src/screens/memory/panes/leaks/leaks_pane.dart

```diff
@@ -148,122 +143,184 @@ class _LeaksPaneState extends State<LeaksPane>
     final json = jsonEncode(task.toJson());
     final jsonFile = _exportController.generateFileName(
       time: now,
-      prefix: _file_prefix,
+      prefix: _Constants.file_prefix,
       extension: 'raw.json',
     );
     return _exportController.downloadFile(json, fileName: jsonFile);
   }
 
+  void dispose() {
+    status.dispose();
+  }
+}
+
+class _LeaksPaneState extends State<LeaksPane>
+    with
+        AutoDisposeMixin,
+        ProvidedControllerMixin<MemoryController, LeaksPane> {
+  late _LeaksPaneController _controller;
```

Comment:
nit: rename this _leaksController, so that _controller (_LeaksPaneController) isn't confused with controller (MemoryController)